README.md

```diff
@@ -106,7 +107,7 @@ You can configure a custom filename template for your file using the query param
 |**`[name]`**|`{String}`|`file.basename`|The basename of the resource|
 |**`[path]`**|`{String}`|`file.dirname`|The path of the resource relative to the `context`|
 |**`[hash]`**|`{String}`|`md5`|The hash of the content, hashes below for more info|
-|**`[N]`**|`{Number}`|``|The `n-th` match obtained from matching the current file name against the query param `regExp`|
+|**`[N]`**|`{String}`, with `N` `{Number}` >0 |``|The `n-th` match obtained from matching the current file name against the `regExp`|
```

Comment:
If it is actually a `{String}` e.g `'4'` instead of a `{Number}`, that should be sufficient for the type column imho. You could add a note in the description that N should be `> 0`

Reply:
No you need to write `[4]`, `[` and `]` decorates the number. And explaining why N is a number > 0 is quite long and already explained on the regExp option. Maybe a link above "`N` `{Number}` >0"?

Reply:
kk, just `{String}` then please :)

Reply:
Oki that's not big deal